src/InvestmentManager.sol

```diff
@@ -472,6 +475,11 @@ contract InvestmentManager is Auth, IInvestmentManager {
         (, lastUpdated) = poolManager.getTrancheTokenPrice(vault_.poolId(), vault_.trancheId(), vault_.asset());
     }
 
+    /// @inheritdoc IInvestmentManager
+    function isGlobalOperator(address, /* vault */ address user) public view returns (bool) {
```

Comment:
Doing it this way has 2 benefits
- Enables us to possibly swap the investment mgr in the future and add other cases for when a contract is an operator of another user, without having to swap the vaults
- Not adding a dependency of the vault on the root